### DISCUSSION TITLE
15th and 16th July : Code is not available on Git Hub

### DISCUSSION THREAD

**Discussion by nandini-naik09** [akash-coded/mern#178] [Q&A]
Hello Akash,

can you please upload the code which we did in 15th and 16th July's class.

Thanks,
Nandini.

**Comment by starvikas1988** [comment]
Hello Akash,
Code for 15th july 2023 topic: Redux explanation and code is missing in github please push it ASP.
Thank You,
Vikas Pandey

**Comment by LNSantosh** [comment]
Hi @akash-coded 
Can you please upload the code for Redux of 9th, 15th, 16th if your check-in issue is resolved. Else you can try add in discussions if you cant push to github.
Thanks,
Santosh L N
